### PR TITLE
Bug/7222 fix hover version number

### PR DESCRIPTION
**Pull request by Sheersendu**
Close: #7222

## PR Details

Previously tooltip was used which showed the message on hover, currently that has been removed hence fixing the issue.
Screen Recording has been attached for reference.

https://github.com/amplication/amplication/assets/62256588/b92c614a-fe3a-430f-b1cb-95dbda12986c

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx [hunk 1]
```diff
@@ -163,27 +163,16 @@ const WorkspaceHeader: React.FC = () => {
               <Icon icon="logo" size="medium" />
             </Link>
           </div>
-          <Tooltip
```
Comment (abrl91) [inline packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx:166]: @Sheersendu the fixed looks good, but there are unrelated changes. Can you please checkout from branch next and create a new PR?

**Review by abrl91 — CHANGES_REQUESTED** [review]
@Sheersendu I left one comment

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx (hunk 1 above)
Comment (Sheersendu) [reply packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx:166]: Hi @abrl91, this change is related to another issue that I was working on. Is it necessary to create e new PR, if I revert back the changes for this file and push will be fine then?

Comment (Sheersendu) [reply packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx]: @abrl91 required changes have been made, hope its fine now :)

**Comment by abrl91** [comment]
@Sheersendu I still see unrelated commits. You need to close this PR, checkout from next and create a new PR

**Comment by Sheersendu** [comment]
As you asked new PR has been raised @abrl91. Link : https://github.com/amplication/amplication/pull/7251
